wkz/file_helper/fit_collector.py

```diff
@@ -105,8 +107,30 @@ def try_to_mount_device():
         else:
             log.warning("could not mount device")
     else:
-        log.warning("Found Garmin device, but could not mount it.")
+        log.warning(f"Found Garmin device, but could not mount it. {mount_output}")
 
 
-def _mount_device_using_gio(bus: str, dev: str) -> str:
-    return subprocess.check_output(["gio", "mount", "-d", f"/dev/bus/usb/{bus}/{dev}"]).decode("utf-8")
+def _mount_device_using_gio(dev: str) -> str:
```

Comment:
Could we maybe rename `dev` to `path` now? Since the argument now holds the path and no longer the device id.
```suggestion
def _mount_device_using_gio(path: str) -> str:
```